pgml-extension/docker/entrypoint.sh

```diff
@@ -18,6 +19,9 @@ echo "Creating user and database..."
 (createdb -U postgres -h 127.0.0.1 pgml_development 2> /dev/null) || true
 psql -d pgml_development -f sql/install.sql -U postgres -h 127.0.0.1 > /dev/null
 
+echo "Installing example datasets"
+psql -d pgml_development -f examples/classification.sql -U postgres -h 127.0.0.1 -f sql/test.sql -P pager
```

Comment:
Why not bring it all in for show and tell?

Reply:
That runs everything.